app/Project.swift

```diff
@@ -26,6 +26,18 @@ let inspectBuildPostAction: ExecutionAction = .executionAction(
     """
 )
 
+let oauthClientIdEnvironmentVariable: EnvironmentVariable = switch Environment.env {
+case .string("staging"): "5339abf2-467c-4690-b816-17246ed149d2"
+case .string("canary"): "5339abf2-467c-4690-b816-17246ed149d2"
```

Comment:
we first need to ship the server PR, so we can get the oauth client ids for the individual environments.